### PR TITLE
feat(jasmine): add matcher transforms that have no direct equivalent in jest, but can be expressed in other ways

### PR DESCRIPTION
**Pull request by jase88**
added support for toBePositiveInfinity, toBeNegativeInfinity, toHaveSize, toHaveBeenCalledOnceWith, toHaveBeenCalledBefore, toHaveSpyInteractions

credits to @puglyfe

### CODE REVIEW

File: src/transformers/expect.test.ts [hunk 1]
```diff
@@ -169,6 +183,10 @@ test('maps expect matchers', () => {
       expect(stuff).not.toHaveBeenCalled();
       expect(stuff).not.toHaveBeenCalled();
       expect(stuff).toHaveBeenCalledWith('foo', 'bar');
+
+      expect(mySpy.mock.calls).toEqual([['foo', 'bar']]);
```
Comment (skovhus) [inline src/transformers/expect.test.ts:187]: Worth double checking this one. I believe it should be `mySpy.mock.calls[0]`, but we miss checking that it is only called once with those arguments.

Comment (puglyfe) [reply src/transformers/expect.test.ts:187]: the nested array implicitly asserts that `mySpy.mock.calls.length === 1`

Comment (skovhus) [reply src/transformers/expect.test.ts:187]: I didn't realize the array was nested. All good then. 👍🏼

**Comment by jase88** [comment]
I think my changes here won't apply for jasmine spec files (at least it didn't work with latest release). I thought `expect.ts` would be some kind of global expect transformation, but it seems to be a transformation for a separate lib? I probably need to move my changes to `jasmine-global.ts`, right?

**Pull request merged**